src/pages/Content/index.js

```diff
@@ -26,3 +32,28 @@ function click(selector) {
   const element = document.querySelector(selector) || null;
   if (element) element.click();
 }
+
+// a function that listens to click events and stores them in the recordedEvents array
+function attachGlobalEventListeners() {
+
+  // capture all click events
+  document.body.addEventListener('click', function (e) {
+
+    // find the best selector for click target (id/class/tag/attr)
+    const selector = finder(e.target);
+
```

Comment:
What if `selector` is not found?

Reply:
Added a try and catch clause, just waiting on confirmation on capturing before I push the changes

Reply:
I've just pushed the try catch block @agamm both here and in #20 so we don't get any conflicts when merged. I've verified that the click listener works great before making this pull request (please refer to the `console.debug()` test within the try block)